platform/lib/routers/growPages.js

```diff
@@ -157,20 +157,12 @@ function rewriteLinks(canonical, html, format) {
 }
 
 // eslint-disable-next-line new-cap
-const pages = express.Router();
+const growPages = express.Router();
 
 const optimizer = AmpOptimizer.create();
 
-pages.get('/*', async (req, res, next) => {
-  // Let known file extensions automatically fallthrough as if they could not
-  // been resolved by the preceeding middleware the pages router can't
-  // resolve them either
-  const KNOWN_FILE_EXTENSIONS = /\.(jpg|png|css|js|map)$/;
-  if (KNOWN_FILE_EXTENSIONS.test(req.path)) {
-    next();
-    return;
-  }
-
+// only match urls with slash at the end or html extension or no extension
+growPages.get(/^(.*\/)?([^\/\.]+|.+\.html|.+\/)$/, async (req, res, next) => {
```

Comment:
This doesn't seem to work for the homepage: https://amp-dev-staging.appspot.com/

Reply:
fixed with https://github.com/ampproject/amp.dev/pull/2612